sql/mob_pools.sql

```diff
@@ -3870,7 +3870,7 @@ INSERT INTO `mob_pools` VALUES (3814,'Suttung','Suttung',135,0x0000B001000000000
 INSERT INTO `mob_pools` VALUES (3815,'Suu_Xicu_the_Cantabile','Suu_Xicu_the_Cantabile',360,0x00003D0400000000000000000000000000000000,10,10,5,240,100,0,1,1,1,2,23,32,428,669,8,0,6,0,0,360,360);
 INSERT INTO `mob_pools` VALUES (3816,'Suzaku','Suzaku',280,0x0000510100000000000000000000000000000000,5,1,3,200,125,0,1,1,0,2,1,32,1079,159,0,1,27,0,0,280,280);
 INSERT INTO `mob_pools` VALUES (3817,'Svelldrake','Svelldrake',266,0x00008F0100000000000000000000000000000000,1,1,3,240,100,0,1,0,0,0,0,0,0,129,0,0,0,0,0,266,266);
-INSERT INTO `mob_pools` VALUES (3818,'Swamfisk','Swamfisk',197,0x00005C0100000000000000000000000000000000,1,1,7,240,100,0,1,0,1,2,0,0,0,3,0,0,0,0,0,4026,197);
+INSERT INTO `mob_pools` VALUES (3818,'Swamfisk','Swamfisk',197,0x00005C0100000000000000000000000000000000,1,1,7,240,100,0,1,0,1,2,0,0,0,3,0,0,0,0,0,1197,197);
```

Comment:
I see you're adjusting an unrelated NM, same below with Barbastelle

Reply:
I did this as I had a merge conflict, so I adjusted their mob skill list ID in mob_skill_lists.sql to reflect this change.